src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/MouseDevice.cs

```diff
@@ -1527,9 +1527,13 @@ private void PreNotifyInput(object sender, NotifyInputEventArgs e)
                                             // If we are over something else (like a raw visual)
                                             // find the containing element.
                                             if (!InputElement.IsValid(mouseOver))
+                                            {
                                                 mouseOver = InputElement.GetContainingInputElement(mouseOver as DependencyObject);
+                                            }
                                             if ((rawMouseOver != null) && !InputElement.IsValid(rawMouseOver))
+                                            {
                                                 rawMouseOver = InputElement.GetContainingInputElement(rawMouseOver as DependencyObject);
+                                            }
                                         }
```

Comment:
Can you provide issue or test method, that cover this piece of code.

Reply:
I don't understand that request, as i only changed the casts.